installing/installing_aws/installing-aws-wavelength-zone.adoc

```diff
@@ -0,0 +1,210 @@
+
+You can install a cluster into an existing Amazon Virtual Private Cloud (VPC) on Amazon Web Services (AWS). The installation program provisions the rest of the required infrastructure, which you can further customize. To customize the installation, modify parameters in the `install-config.yaml` file before you install the cluster.
+
+Installing a cluster on AWS into an existing VPC requires extending compute nodes to the edge of the Cloud Infrastructure by using AWS {zone-type}.
+
+You can use a provided CloudFormation template to create network resources. Additionally, you can modify a template to customize your infrastructure or use the information that they contain to create AWS resources according to your company's policies.
+
+[IMPORTANT]
+====
+The steps for performing an installer-provisioned infrastructure installation are provided for example purposes only. Installing a cluster in an existing VPC requires that you have knowledge of the cloud provider and the installation process of {product-title}. You can use a CloudFormation template to assist you with completing these steps or to help model your own cluster installation. Instead of using the CloudFormation template to create resources, you can decide to use other methods for generating these resources.
+====
+
+// Creating a VPC in AWS
+include::modules/installation-creating-aws-vpc-localzone.adoc[leveloffset=+2]
+
+// CloudFormation template for the VPC
+include::modules/installation-cloudformation-vpc-localzone.adoc[leveloffset=+2]
+
+// Creating a VPC Carrier Gateway
+include::modules/installation-creating-aws-vpc-carrier-gw.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See link:https://aws.amazon.com/s3/[Amazon S3] in the AWS documentation.
+
+// CloudFormation template for the VPC Carrier Gateway
+include::modules/installation-cloudformation-vpc-carrier-gw.adoc[leveloffset=+2]
+
+// Creating subnets in Wavelength Zones
+include::modules/installation-creating-aws-vpc-subnets-wz.adoc[leveloffset=+2]
+
+// CloudFormation template for the subnet that uses AWS Wavelength Zones
+include::modules/installation-cloudformation-subnet-localzone.adoc[leveloffset=+2]
+
+// Modifying an installation configuration file to use AWS Wavelength Zones subnets
+include::modules/installing-aws-edge-zones-custom-vpc-config.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about viewing the CloudFormation stacks that you created, see link:https://console.aws.amazon.com/cloudformation[AWS CloudFormation console].
+* For more information about AWS profile and credential configuration, see link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[Configuration and credential file settings] in the AWS documentation.
+
+.Next steps
+* xref:../../installing/installing_aws/installing-aws-wavelength-zone.adoc#installation-launching-installer_installing-aws-wavelength-zone[Deploying the cluster]
```

Comment:
NIT: for some reason this block ("Additional resources" and "next steps") are missing in the Local Zone Day-2 page (I can see in WL):
- LZ: https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone#installing-aws-edge-zones-custom-vpc-config_installing-aws-localzone
- WL: https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-wavelength-zone#installing-aws-edge-zones-custom-vpc-config_installing-aws-wavelength-zone

Reply:
Updated